src/every_query/eval_suite/gen_index_times.py

```
@@ -0,0 +1,77 @@
+    return hash_hex
+
+
+@hydra.main(config_path="./conf", config_name="gen_index_times_config", version_base=None)
+def main(cfg: DictConfig) -> None: 
+    
+    task_dir = Path(cfg.io.task_dir)
+    read_dir = task_dir / "all"
+    out_root = Path(cfg.io.out_root)
+
+    split = held_out_split             
+    method = "random"
```

Ruff (0.14.8)

Local variable `method` is assigned to but never used

Remove assignment to unused variable `method`

(F841)